pkg/apis/operator/v1alpha1/tektonchain_validation.go

```diff
@@ -25,7 +25,13 @@ import (
 	"knative.dev/pkg/apis"
 )
 
-var allowedArtifactsStorage = sets.NewString("tekton", "oci", "gcs", "docdb", "grafeas", "kafka")
+var (
+	allowedArtifactsTaskRunFormat     = sets.NewString("in-toto", "slsa/v1", "")
+	allowedArtifactsPipelineRunFormat = sets.NewString("in-toto", "slsa/v1", "")
+	allowedArtifactsStorage           = sets.NewString("tekton", "oci", "gcs", "docdb", "grafeas", "kafka")
+	allowedX509SignerFulcioProvider   = sets.NewString("google", "spiffe", "github", "filesystem", "")
+	allowedTransparencyConfigEnabled  = sets.NewString("true", "false", "manual", "")
+)
```

Comment:
in all sets add the empty string `""` as a first value. will be more readable
and move `allowedArtifactsStorage` to last on the var list, as it do not contain `""` value

Reply:
done !